lab-dan/routes/dog-routes.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
+
+const responseHandler = require('../lib/responseHandler')
+
+module.exports = (router, storage) => {
+
+  // router expects 3 different things: verb, route, and callback
+  // one callback per verb/route combo
+  router.get('/', function(request, response) {
+    responseHandler.sendText(response, 200, 'Hello, world! This is the amazing dogs api\n')
```

Comment:
At this point, you have Express to handle responses, so you can refactor (remove the responseHandler module), and utilize the response functionality via express.